modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
+
+[discrete]
+== Purpose
+
+etcd certificates are signed by the etcd-signer; they come from a certificate
+authority (CA) that is generated by the bootstrap process.
+
+[discrete]
+== Location
+
+* CA certificates:
+** etcd CA certificate: `/etc/ssl/etcd/ca.crt`
+** etcd metric CA certificate: `/etc/ssl/etcd/metric-ca.crt`
+* Server certificates: `/etc/ssl/etcd/system:etcd-server`
+* Client certificates: `<api_server_pod_directory>/secrets/etcd-client/`
+* Peer certificates: `/etc/ssl/etcd/system:etcd-peer`
+* Metric certificates: `/etc/ssl/etcd/metric-signer`
+
+[discrete]
+== Expiration
+
+The CA certificates are valid for 10 years. The peer, client, and server
+certificates are valid for three years.
+
+[discrete]
+== Management
+
+These certificates are managed by the system and not the user.
+
+[discrete]
+== Services
+
+etcd certificates are used for encrypted communication between etcd member
+peers, as well as encrypted client traffic. The following certificates are
+generated and used by etcd and other processes that communicate with etcd:
+
+* Peer certificates: Used for communication between etcd members.
+* Client certificates: Used for encrypted server-client communication. Client
+certificates are currently used by the API server only, and no other service
+should connect to etcd directly except for the proxy.
```

Comment:
@hexfusion @ahardin-rh  looks like the client secrets (etcd-client, etcd-metric-client, etcd-metric-signer, etcd-signer) are added to the `openshift-config` namespace and the `openshift-monitoring` namespace in addition to the `openshift-kube-apiserver` namespace.  

Should we note that here?  

/cc @s-urbaniak (`kube-etcd-client-certs` is what I'm referring to)